components/cam/src/physics/crm/camrt/radiation.F90

```diff
@@ -0,0 +1,1641 @@
+
```

Comment:
I think we should drop this file and folder. We are not planning to support CAMRT at all.